src/Symfony/Component/Notifier/Bridge/Gitter/GitterTransport.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Gitter;
+
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Notifier\Exception\TransportException;
+use Symfony\Component\Notifier\Exception\UnsupportedMessageTypeException;
+use Symfony\Component\Notifier\Message\ChatMessage;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SentMessage;
+use Symfony\Component\Notifier\Transport\AbstractTransport;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author Christin Gruber <c.gruber@touchdesign.de>
+ */
+final class GitterTransport extends AbstractTransport
+{
+    protected const HOST = 'api.gitter.im';
+
+    private $token;
+    private $room;
+    private $path;
```

Comment:
Based on https://developer.gitter.im/docs/rest-api I am not convinced that a path is really supported.
If I am right
* [ ] the property can be removed
* [ ] it does not need to be get/set in `GitterTransportFactory`
* [ ] the `getEndpoint` method does not need to be overridden
* [ ] the related tests could be removed

Reply:
I have dropped the path property :+1: